NoClassDefFoundError regarding Python SDK class when generating tests with Spring configuration

**Description**

NoClassDefFoundError `com/jetbrains/python/sdk/BasePySdkExtKt` from `UtTestsDialogProcessor.kt`
when generating tests with Spring XML configuration

**To Reproduce**

1. Run a Spring based project in IntelliJ Idea 2023.1 Ultimate (for example spring-petclinic)
2. Install [IU plugin built from main](https://github.com/UnitTestBot/UTBotJava/actions/runs/4680658298)
3. Try to generate tests with any Spring configuration chosen

**Expected behavior**

No exceptions are expected to be thrown down into IDE.

**Actual behavior**

NoClassDefFoundError 

**Visual proofs (screenshots, logs, images)**

~~~java
java.lang.NoClassDefFoundError: com/jetbrains/python/sdk/BasePySdkExtKt
	at org.utbot.intellij.plugin.generator.UtTestsDialogProcessor$createTests$1$1.run(UtTestsDialogProcessor.kt:241)
	at com.intellij.openapi.progress.impl.CoreProgressManager.startTask(CoreProgressManager.java:429)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.startTask(ProgressManagerImpl.java:114)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcessWithProgressAsynchronously$6(CoreProgressManager.java:480)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$3(ProgressRunner.java:252)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:186)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$13(CoreProgressManager.java:604)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:679)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:635)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:603)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:60)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:173)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$4(ProgressRunner.java:252)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:702)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:699)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:699)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.ClassNotFoundException: com.jetbrains.python.sdk.BasePySdkExtKt PluginClassLoader(plugin=PluginDescriptor(name=UnitTestBot, id=org.utbot.intellij.plugin.id, descriptorPath=plugin.xml, path=~\AppData\Roaming\JetBrains\IntelliJIdea2023.1\plugins\utbot-intellij, version=2023.4.4262., package=null, isBundled=false), packagePrefix=null, instanceId=399, state=active)
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.kt:150)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
	... 21 more
~~~

**Environment**

Windows 10 Pro
IntelliJ IDEA Ultimate 2023.1

--- a/utbot-intellij/src/main/kotlin/org/utbot/intellij/plugin/generator/UtTestsDialogProcessor.kt
+++ b/utbot-intellij/src/main/kotlin/org/utbot/intellij/plugin/generator/UtTestsDialogProcessor.kt
@@ -8,6 +8,7 @@ import com.intellij.openapi.progress.ProgressIndicator
 import com.intellij.openapi.progress.Task
 import com.intellij.openapi.project.DumbService
 import com.intellij.openapi.project.Project
+import com.intellij.openapi.roots.ModuleRootManager
 import com.intellij.openapi.roots.OrderEnumerator
 import com.intellij.openapi.roots.ProjectFileIndex
 import com.intellij.openapi.ui.Messages
@@ -27,7 +28,6 @@ import com.intellij.task.impl.ProjectTaskList
 import com.intellij.util.concurrency.AppExecutorUtil
 import com.intellij.util.containers.ContainerUtil
 import com.intellij.util.containers.nullize
-import com.jetbrains.python.sdk.basePath
 import java.io.File
 import java.nio.file.Path
 import java.nio.file.Paths
@@ -236,13 +236,17 @@ object UtTestsDialogProcessor {
                                         when (val approach = model.typeReplacementApproach) {
                                             DoNotReplace -> emptyList()
                                             is ReplaceIfPossible -> {
-                                                // TODO: use common parent path for srcModule and used Spring
-                                                //  config module if they are different modules
-                                                val projectFileStorage = model.srcModule.basePath
+                                                val contentRoots = listOfNotNull(
+                                                    model.srcModule,
+                                                    springConfigClass?.module
+                                                ).distinct().flatMap { module ->
+                                                    ModuleRootManager.getInstance(module).contentRoots.toList()
+                                                }
                                                 process.getSpringBeanQualifiedNames(
                                                     classpathForClassLoader,
                                                     approach.config,
-                                                    projectFileStorage,
+                                                    // TODO: consider passing it as an array
+                                                    contentRoots.joinToString(File.pathSeparator),
                                                 )
                                             }
                                         }

